app/models/manageiq/providers/openstack/cloud_manager/refresher.rb

```diff
@@ -6,6 +6,13 @@ def parse_inventory(ems, _targets)
       ManageIQ::Providers::Openstack::CloudManager::RefreshParser.ems_inv_to_hashes(ems, refresher_options)
     end
 
+    # TODO(lsmola) NetworkManager, remove this once we have a full representation of the NetworkManager.
+    # NetworkManager should refresh base on it;s own conditions
```

Comment:
Minor typo with the semicolon...just remove it because it shouldn't be a quote either.